src/iexamine.cpp

```diff
@@ -3530,29 +3574,28 @@ void iexamine::autodoc( player &p, const tripoint &examp )
             const itype *itemtype = it->type;
             const bionic_id &bid = itemtype->bionic->id;
 
-            if( p.has_bionic( bid ) ) {
-                popup( _( "You have already installed this bionic."  ) );
+	    installer.add_msg_player_or_npc(
+                _( "You prepare to install the %1$s on %2$s." ),
+                _( "<npcname> prepares to install the %1$s on %2$s." ),
+                it->display_name().c_str(), patient.name );
+            if( patient.has_bionic( bid ) ) {
+                popup( _( "%1$s already installed this bionic."  ), pronoun );
                 return;
-            } else if( bid->upgraded_bionic && !p.has_bionic( bid->upgraded_bionic ) ) {
-                popup( _( "You have no base version of this bionic to upgrade." ) );
+            } else if( bid->upgraded_bionic && !patient.has_bionic( bid->upgraded_bionic ) ) {
+                popup( _( "%1$s no base version of this bionic to upgrade." ), pronoun );
```

Comment:
Please add translation comments here about what this `%1$s` here and below supposed to mean.